Sources/Sentry/SentryCrashIntegration.m

```diff
@@ -131,7 +131,19 @@ - (void)configureScope
             [outerScope setContextValue:appData forKey:@"app"];
 
             [outerScope addScopeListener:^(SentryScope *_Nonnull scope) {
-                [SentryCrash.sharedInstance setUserInfo:[scope serialize]];
```

Comment:
We're we adding more stuff than we are now? Here looks like the whole scope was added

Reply:
I missed the line bellow